### PR TITLE
FB-706

### PR DESCRIPTION
**Pull request by sequencemedialimited**
Addresses the Components Core changes described in [FB-706](https://dsdmoj.atlassian.net/browse/FB-706).

- Components Core package has a dependency on `govuk-frontend` at NPM and GitHub
- Components Core contains static CSS files from the [GOV.UK Frontend GitHub repository](https://github.com/alphagov/govuk-frontend) which are _not_ contained in `govuk-frontend` NPM module

I have manually updated the `/assets` folder in Components Core with the correct versions of the static CSS from the [GOV.UK Frontend GitHub repository](https://github.com/alphagov/govuk-frontend).

I have modified the Nunjucks templates such that they refer to those files explicitly.

I have also changed the version number of _Components Core_ to track `govuk-frontend` (as it did previously).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ministryofjustice/fb-components-core",
-  "version": "3.1.1",
+  "version": "3.3.0",
```
Comment (sequencemedialimited) [inline package.json:3]: This change tracks `govuk-frontend`

File: templates/nunjucks/_base.njk.html [hunk 2]
```diff
@@ -3,9 +3,9 @@
 {% block pageTitle %}{% if page.errorList %}Error: {% endif %}{% if page.heading %}{{ page.heading }}{% else %}{% if page.components[0].label.html %}{{ page.components[0].label.html }}{% else %}{{ page.components[0].legend.html }}{% endif %}{% endif %}{% if page._id != 'page.start' %} - {{ page.header.serviceName }}{% endif %} - GOV.UK{% endblock %}
 
 {% block head %}
-<link rel="stylesheet" href="/assets/govuk-frontend-{{govuk_frontend_version}}.min.css">
+<link rel="stylesheet" href="/assets/govuk-frontend-3.3.0.min.css">
```
Comment (sequencemedialimited) [inline templates/nunjucks/_base.njk.html:6]: These are explicit references to files contained in the `assets` folder at the root of this project

Other files (like fonts and icons) will be served from that folder too

Comment (sequencemedialimited) [reply templates/nunjucks/_base.njk.html:6]: (They are manually added from the GOV.UK Frontend repository on GitHub)

**Review by cob16 — APPROVED** [review]
looks good just 1 repo to chose what version to use :+1: 

looking forward to https://dsdmoj.atlassian.net/browse/FB-710 as well

**Comment by sequencemedialimited** [comment]
I can do https://dsdmoj.atlassian.net/browse/FB-710 pretty quickly based on code I've written a thousand times, but I'm going to bump it until I've got some of these bugs out of the way, and then make a stronger case for it -- at the moment I haven't convinced _myself_

**Pull request merged**